docs/charts/bar.md

```diff
@@ -6,17 +6,17 @@ A bar chart provides a way of showing data values represented as vertical bars.
     "type": "bar",
     "data": {
         "labels": [
-            "January",
-            "February",
-            "March",
-            "April",
-            "May",
-            "June",
+            "January", 
```

Comment:
Please remove the spaces that were added to the end of each line

Reply:
done, that's problem of my phpStorm, will change settings, sorry for troubling with whitespaces this time (

Reply:
I wanted to check on this because you said "done', but it doesn't look like the whitespaces have been removed yet

Reply:
hm, just fixed it online via github tool. Seems those whitespaces were existed. Still blaming my soft for this, sorry